module2/homework/insensitivePalindrom/palindrome.cpp

```diff
@@ -0,0 +1,12 @@
+#include "palindrome.hpp"
+#include <algorithm>
+
+
+bool is_palindrome(std::string str){
+
+    str.erase(std::remove_if(str.begin(), str.end(), [](char letter) { return std::isalpha(letter) == 0; }), str.end());
```

Comment:
It can be done a little bit different
```suggestion
    str.erase(std::remove_if(str.begin(), str.end(), [](char letter) { return !std::isalpha(letter); }), str.end());
```

Reply:
+1

Reply:
+1

Reply:
it doesn't work in this way. it should be false.

Reply:
ok, I didn't see exclamation mark :)